pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "geoapps"
-version = "0.11.0-beta.2"
+version = "0.11.0-rc1"
```

use format compliant with Semver
```suggestion
version = "0.11.0-rc.1"
```